### DISCUSSION TITLE
Veronica/Disney XD Netherlands

### DISCUSSION THREAD

**Discussion by IseyJay** [iptv-org/community#934] [Q&A]
### Channel ID

VeronicaDisneyXD.nl

### Notes (optional)

Disney XD Netherlands Is The Second Disney XD Channel That Still Broadcasts In Europe (Along With Poland)
The Channel Runs From Around 4am To 6pm as well as being timeshared with Veronica TV Through Out 6pm to around 2am

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by kaikyoofers** [comment]
Is that a xtream link? We dont allow that, @freearhey please remove the valid stream tag

**Comment by BellezaEmporium** [comment]
> Is that a xtream link? We dont allow that, @freearhey please remove the valid stream tag

Yep, that's from an Xtream server.

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19853
